controller/scheduler-v2/scheduler.go

```diff
@@ -101,6 +101,8 @@ func (s *Scheduler) Sync() (err error) {
 		s.sendEvent(NewEvent(EventTypeClusterSync, err, nil))
 	}()
 
+	s.SyncFormations()
```

Comment:
I think we should be triggering `SyncFormations` less often than `Sync` (and maybe call the latter `SyncJobs`), e.g. in `Scheduler.Run`:

```go
jobSync := time.Tick(30 * time.Second)
formationSync := time.Tick(time.Minute)

for {
	select {
	case <-jobSync:
		s.SyncJobs()
	case <-formationSync:
		s.SyncFormations()
	default:
	}

	// code for starting jobs
}
```